pages/to_do_list_page.ts

```diff
@@ -0,0 +1,43 @@
+import { test, Page, expect } from '@playwright/test';
+
+export class ToDoListPage {
+    readonly page: Page;
+
+    constructor(page: Page) {
+        this.page = page;
+    }
+    
+
+    async open() {
+        await this.page.goto('https://webdriveruniversity.com/To-Do-List/index.html');
```

Comment:
You hardcoded url. Create global readonly variable.

Reply:
resolved